packages/dynamite/dynamite/lib/src/models/openapi/schema.dart

```diff
@@ -16,8 +17,37 @@ abstract class Schema implements Built<Schema, SchemaBuilder> {
 
   static Serializer<Schema> get serializer => _$schemaSerializer;
 
+  @BuiltValueField(wireName: r'$id')
+  Uri? get id;
+
   @BuiltValueField(wireName: r'$ref')
-  String? get ref;
+  Uri? get ref;
+
+  Schema buildRef(Map<String, dynamic> json) {
```

Comment:
```suggestion
  Schema resolveRef(Map<String, dynamic> json) {
```
I think this better reflects what it does. The ref isn't built, the ref is resolved and then the schema for it is built